site/static/shared.js

```diff
@@ -252,3 +252,14 @@ function make_request(path, body) {
         console.log("error fetching ", path, ": ", err);
     });
 }
+
+// https://stackoverflow.com/questions/6234773
+// handmade escape, as we don't use any libs (why?)
```

Comment:
It's good to ask why we don't use any libs, but I don't think it's necessary to pose this in a comment.